packages/flutter/lib/src/semantics/semantics_service.dart

```diff
@@ -29,8 +29,12 @@ class SemanticsService {
   ///
   /// For example a camera application can use this method to make accessibility
   /// announcements regarding objects in the viewfinder.
-  static Future<void> announce(String message, TextDirection textDirection) async {
-    final AnnounceSemanticsEvent event = AnnounceSemanticsEvent(message, textDirection);
+  ///
+  /// The assertiveness level of the announcement can be determined by [assertiveness].
```

Comment:
nit: I'd replace "can be" with "is"

Reply:
This paragraph does not add any more information other than the parameter name.

Maybe mention why developer would want to use this parameter? maybe give an example situation and which enum value you can use?